src/report_error.ml

```diff
@@ -89,6 +89,13 @@ let report_with_backtrace exn =
 
 let reported = ref String.Set.empty
 
+let error_table = Hashtbl.create 1024
+
+let print_all_errors_to_console () =
```

Comment:
Currently this function just allows one to print in the order that the error messages were hashed.  Perhaps it might be beneficial to allow individuals to print errors sorted by location?

Reply:
Yes. I think ideally we want an order that respect the compilation DAG. For instance if you have two libraries X and Y and Y depend on X, it'd be a bit weird to see Y errors before X errors.